k8s/logs-spack-io/deployments.yaml

```diff
@@ -0,0 +1,352 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: logs-spack-io-redirect
```

Comment:
I'm assuming you pulled the `-redirect` part of this name from the spack.io config?  If so, I would skip all of the redirect stuff, unless this service is actually redirecting traffic somewhere?

Reply:
So, `logs-spack-io` would be a better name.